product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/18_dbms_xmldom/getchildnodes.mdx

```diff
@@ -12,11 +12,11 @@ GETCHILDNODES(n DOMNode) RETURN DOMNodeList
 
 `n`
 
-The `DOMNode` whose childnode list is to be retrieved.
+The `DOMNode` whose child node list to retrieve.
 
 ## Examples
 
-This example executes a function named `func1` that creates the XML structure `<Deptartments>Dept1</Deptartments>` and returns the root node which is a `DOMDocument`.
+This example executes a function `func1` that creates the XML structure `<Departments>Dept1</Departments>` and returns the root node, which is a `DOMDocument`.
```

Comment:
This looked to me like a typo, but please verify. It appears in several spots.

Reply:
It's definitely a typo. Thanks for spotting this! 
I think we should probably do a search and replace to catch instances in code snippets. Unless you already did it? Let me know :)

Reply:
That was a typo, but fixing it just in the comments does not suffice. The code snippets should also be changed and tested.

Reply:
Did a search + replace to catch all typos ✅